utils/models_to_deprecate.py

```diff
@@ -124,19 +128,21 @@ def get_list_of_models_to_deprecate(
         }
 
         # We make successive calls to the hub, filtering based on the model tags
-        n_seen = 0
-        for model, model_info in models_info.items():
+        print("Making calls to the hub to find models below the threshold number of downloads")
+        num_models = len(models_info)
+        for i, (model, model_info) in enumerate(models_info.items()):
+            if max_num_models != -1 and i > max_num_models:
+                break
```

Comment:
I think this is redundant: it's already controlled when we build `models_info` at the beginning